portfolio/src/main/webapp/index.html

```diff
@@ -4,25 +4,32 @@
     <meta charset="UTF-8">
     <title>Lily Zhou</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
+    <div class="content">
+      <img id="pf-image" src="/images/flower.jpg" alt="Red Flower">
       <h1>Lily Zhou</h1>
       <div class="icon-row">
         <a href="https://www.linkedin.com/in/lily-z-408069122"><img src="/images/linkedin.jpg" width="30" height="30" alt="LinkedIn"></a>
-        <a href="https://github.com/lilymzhou"><img src="/images/github.jpg" width="30" height="30" alt="GitHub"></a>
+        <a href="https://github.com/lilymzhou"><img src="/images/github.svg" width="30" height="30" alt="GitHub"></a>
+      </div>
+      <div id="intro-box">
```

Comment:
prefer using classes instead of ids for styling, since this allows you to reuse styles across different elements